src/WithBackgroundImage/index.js

```diff
@@ -1,26 +1,67 @@
-import React from 'react';
+import React, {useEffect} from 'react';
 import PropTypes from 'prop-types';
 
 import useBaseUrl from '@docusaurus/useBaseUrl';
 import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
 import useThemeContext from '@theme/hooks/useThemeContext';
 
+import classnames from 'classnames';
+import styles from './styles.module.css';
+
+const getBackgroundImages = (imageDark, imageDarkHover, imageLight, imageLightHover) => {
```

Comment:
Please add a note to simplify this. I would like to store all theme data somewhere else. This component should only take an image and hover state. Something higher up will determine what those props are based on the chosen theme.

Reply:
We should sync tomorrow on this